ref/Image_Manipulation_and_Rendering.md

```diff
@@ -314,6 +312,35 @@ It plots a point with the corresponding color and opacity and by this creates a
 
 ------
 
+#### Getting pixel data: `imagergb(‹imagename›,‹int›,‹int›)`
+
+**Description:**
+This function does the same as `imagergba(‹imagename›,‹int›,‹int›)`. In particular, it also delivers the alpha value in order to preserve backward compatibility.
+
```

Comment:
I wonder whether it would make sense to have a modifier to enable more modern behavior instead of this compatibility burden. Probably not, though, as `_(1..3)` is shorter than any modifier `, modern->true`.